File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/response/type/ConnectNodeAppendedImpulse.kt
package gaia.sdk.response.type

import gaia.sdk.Uuid
import gaia.sdk.ISO8601
import gaia.sdk.Struct
import gaia.sdk.response.intf.*
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonCreator
import gaia.sdk.request.enumeration.*

/**
* Impulse which indicates the result appending a node connection
*/
data class ConnectNodeAppendedImpulse @JsonCreator constructor(
    @JsonProperty("id") val id:Uuid? = null, 
    /**
    * all edges that are set after this request
    */
    @JsonProperty("allEdges") val allEdges:List<Edge>? = null, 
    /**
    * edge created by this request
    */
    @JsonProperty("newEdge") val newEdge:Edge? = null
)

File: gaia-sdk-python/gaia_sdk/graphql/response/type/ConnectNodeAppendedImpulse.py

from gaia_sdk.graphql.response.type.Edge import Edge

Uuid = str
String = str
ISO8601 = str
Struct = dict
Float = float
Int = int
Boolean = bool


class ConnectNodeAppendedImpulse:
    """
    Impulse which indicates the result appending a node connection
    """
    dictionary: dict

    def __init__(self, dictionary: dict):
        self.dictionary = dictionary

    def __eq__(self, other):
        if type(other) is type(self):
            return self.dictionary == other.dictionary
        return False

    def __repr__(self):
        return {'dictionary': self.dictionary}

    @property
    def id(self) -> Uuid:
        return Uuid(self.dictionary.get("id"))
    """
    edge created by this request
    """
    @property
    def new_edge(self) -> Edge:
        return Edge(self.dictionary.get("newEdge"))
